lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -264,6 +264,12 @@ def get(self, request, *args, **kwargs):
             if allow_public:
                 handouts_html = get_course_info_section(request, request.user, course, 'handouts')
 
+        elif is_masquerading(request.user, course_key):
```

Comment:
Nit: could have a small performance improvement to this call by passing in the masquerade object.